frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -1496,3 +1214,416 @@ function MeasurementToolSwitch({ activeTool }: { activeTool: AnnotationTool }) {
     </Radio.Group>
   );
 }
+
+function MoveTool() {
```

Comment:
the code below was merely extracted.

Reply:
yes, I'd like to do this too, but thought that the PR would become even harder to review :see_no_evil: now that you have already reviewed the PR, I could move the code around and during review of newer commits you could simply skip over that commit? what do you think?

Reply:
sounds great :+1:

Reply:
I just pushed a commit that splits toolbar_view into several modules and moves them into a common folder. no need to review as I merely moved code around and adapted the imports.